specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-11-16-preview/examples/PostAccessReviewHistoryDefinition.json

```diff
@@ -0,0 +1,81 @@
+        "type": "Microsoft.Authorization/accessReviewHistoryDefinition",
+        "properties": {
+          "displayName": "Hello world name",
+          "reviewHistoryPeriodStartDateTime": "2021-10-01T00:00:00-07:00",
+          "reviewHistoryPeriodEndDateTime": "2021-12-01T00:00:00-08:00",
+          "decisions": [
+            "Approve",
+            "Deny",
+            "NotReviewed",
+            "DontKnow",
+            "NotNotified"
+          ],
+          "status": "Requested",
+          "createdDateTime": "2021-12-23T00:27:33.6905189+00:00",
+          "createdBy": {
+            "principalId": "673ad0d8-7b0e-4201-aaeb-74cdcbf22af9",
+            "principalType": null,
+            "principalName": "levi",
+            "userPrincipalName": "levi"
+          },
+          "scopes": [
+            {
+              "resourceId": "/subscriptions/129a304b-4aea-4b86-a9f7-ba7e2b23737a",
+              "roleDefinitionId": "/subscriptions/129a304b-4aea-4b86-a9f7-ba7e2b23737a/providers/Microsoft.Authorization/roleDefinitions",
+              "principalType": "user",
+              "assignmentState": null,
+              "inactiveDuration": null,
+              "expandNestedMemberships": true
+            },
+            {
+              "resourceId": "/subscriptions/129a304b-4aea-4b86-a9f7-ba7e2b23737a",
+              "roleDefinitionId": "/subscriptions/129a304b-4aea-4b86-a9f7-ba7e2b23737a/providers/Microsoft.Authorization/roleDefinitions/1562cf42-00b9-457c-86ef-5702d4132904",
+              "principalType": "user",
+              "assignmentState": null,
+              "inactiveDuration": null,
+              "expandNestedMemberships": false
+            }
+          ],
+          "settings": {
+            "range": {
+              "type": "endDate",
+              "numberOfOccurrences": 1,
+              "startDate": "2021-12-01T00:00:00-08:00",
+              "endDate": "2023-12-01T00:00:00-08:00"
+            }
+          },
+          "instances": [
+            {
+              "id": "44724910-d7a5-4c29-b28f-db73e717165a",
+              "name": "44724910-d7a5-4c29-b28f-db73e717165a",
+              "type": "Microsoft.Authorization/accessReviewHistoryInstance",
+              "properties": {
+                "reviewHistoryPeriodStartDateTime": "2021-10-01T00:00:00-07:00",
+                "reviewHistoryPeriodEndDateTime": "2021-12-01T00:00:00-08:00",
+                "displayName": "Hello world name",
+                "status": "Requested",
+                "runDateTime": "2021-12-23T00:27:33.6905189+00:00",
+                "fulfilledDateTime": "2021-12-23T00:28:33.6905189+00:00",
+                "downloadUri": "https://dfermconsolreportusc.blob.core.windows.net/df-erm-reports/Hello world name-22be232e-a93d-42a3-8ac5-313cfd29a0eb.csv?sv=2015-04-05&ss=b&srt=o&sp=rl&st=2021-03-22T19:39:38.0000000Z&se=2021-03-23T19:41:38.0000000Z&spr=https&sig=84rlGCIgU4ToMn%2FFLncBXq95O8a8RsFlwQY1Knl%2Fo%2FI%3D",
```

Comment:
>downloadUri [](http://example.com/codeflow?start=17&length=11)

You are executing a POST API to return a resource properties and expect users to use `downloadUri` property? Thinking about automation, how does a user know which instance - downloadUri to use? I guess I am not following the design. I was thinking that you execute a POST API and the return is just this downloadUri value.

Is the downloadUri per instance? If yes, then the POST API must be related to an instance resource id so you can return download uri.